apps/payments/wise.py

```diff
@@ -33,21 +32,11 @@ def wise_webhook():
         request.headers,
     )
 
+    environment = app.config["TRANSFERWISE_ENVIRONMENT"]
     try:
-        b64decode(request.headers["X-Signature-SHA256"])
-        if request.json is None:
-            raise ValueError("Request does not contain JSON")
+        validate_request(request=request, environment=environment)
     except Exception as e:
-        logger.info("Unable to parse Wise webhook request")
-        abort(400)
-
-    valid_signature = verify_signature(
-        request.data,
-        request.headers["X-Signature-SHA256"],
-        app.config["TRANSFERWISE_ENVIRONMENT"],
-    )
-    if not valid_signature:
-        logger.exception("Error verifying Wise webhook signature")
```

Comment:
Ah, I hadn't spotted this `exception`-level log line.  Making some adjustments for this in a few moments.